content/docs/reactivesearch/flutter-searchbox/quickstart.md

```diff
@@ -118,6 +118,19 @@ class HomePage extends StatelessWidget {
                           {'field': 'original_title', 'weight': 1},
                           {'field': 'original_title.search', 'weight': 3}
                         ],
+						// This prop is used to return only the distinct value documents for the specified field
+						distinctField: 'authors.keyword',
+						// This prop allows specifying additional options to the distinctField prop
+                        distinctFieldConfig: {
```

Comment:
@anik-ghosh-au7 Spacing issues

Reply:
@bietkul  I have formatted the code using dart formatted and pasted it here